{"name": ["shgo_slsqp_sobol_cube", "shgo_powell_sobol_cube", "shgo_nelder_sobol_cube", "scipy_slsqp_cube", "scipy_powell_cube", "scipy_nelder_cube", "scipy_lbfgsb_cube", "pysot_ei_cube", "pysot_lcb_cube", "pysot_random_cube", "pysot_srbf_cube", "pysot_dycors_cube", "ax_default_cube", "optuna_cmaes_cube", "optuna_tpe_cube", "optuna_random_cube", "optuna_random_cube_clone", "optuna_random_cube_clone_1", "optuna_random_cube_clone_2", "platypus_genetic_cube", "platypus_evolutionary_cube", "platypus_nsgaii_cube", "platypus_cmaes_cube", "platypus_gde3_cube", "platypus_ibea_cube", "platypus_moead_cube", "platypus_omopso_cube", "platypus_smpso_cube", "platypus_spea2_cube", "platypus_epsmoea_cube", "nevergrad_ngopt_cube", "nevergrad_ngopt4_cube", "nevergrad_de_cube", "nevergrad_portfolio_cube", "nevergrad_oneplus_cube", "nevergrad_hammersley_cube", "nevergrad_ngopt8_cube", "hyperopt_tpe_cube", "hyperopt_atpe_cube", "hyperopt_rand_cube", "pymoo_nelder_cube", "pymoo_nsga3_cube", "pymoo_unsga3_cube", "pymoo_pattern_cube", "pymoo_nsga2_cube", "pymoo_unsga3_cube", "skopt_gp_default_cube", "ultraopt_random_cube", "ultraopt_forest_cube", "ultraopt_gbrt_cube", "ultraopt_etpe_cube", "bayesopt_ucb_cube", "bayesopt_ei_cube", "bayesopt_poi_cube", "dlib_cube", "dlib_default_cube", "nlopt_direct_cube", "nlopt_ags_cube", "nlopt_esch_cube", "nlopt_isres_cube", "nlopt_crs2lm_cube", "nlopt_directr_cube", "nlopt_directo_cube", "nlopt_directl_cube", "bobyqa_default_cube", "bobyqa_noise_cube", "hebo_cube_sequential", "hebo_cube_batch", "hebo_sequential_cube", "hebo_batch_cube"], "count": [55, 76, 87, 133, 4, 17, 2, 242, 207, 232, 214, 250, 207, 205, 226, 227, 236, 214, 273, 0, 0, 0, 0, 0, 0, 185, 0, 0, 0, 0, 230, 246, 241, 232, 235, 236, 261, 220, 238, 254, 209, 232, 473, 235, 0, 0, 199, 98, 99, 92, 103, 207, 230, 223, 85, 102, 219, 222, 201, 236, 236, 216, 220, 209, 222, 202, 33, 42, 7, 11], "rating": [1362.745276973245, 1279.4278896551857, 1347.3784055390736, 1503.7019097266868, 1498.0453045972488, 1382.9233430604934, 1611.470037332866, 2222.9635223350997, 1946.096758237845, 1089.150916634099, 2509.0698536220766, 2582.683125999819, 2427.73733970044, 1588.7728152400532, 1784.0982180921842, 1600, 1600, 1600, 1600, 1600, 1600, 1600, 1600, 1600, 1600, 724.9288276859254, 1600, 1600, 1600, 1600, 1864.6366841845754, 2123.9373777901155, 1222.7015247276531, 1020.5546447643576, 2264.0631547665903, 1095.4692106415077, 2050.0885901011407, 1397.3293511270383, 1565.8124031871312, 1587.378909017942, 1208.889013053926, 990.0980146129973, 863.7294947639646, 2009.904252446619, 1600, 1600, 2332.5096217917667, 1414.8785784894478, 1933.3345376796763, 1735.118744629652, 2014.8934977958068, 2309.6035961287216, 2317.4195648116543, 2347.332313618297, 2231.35323295821, 2031.7358268689015, 1703.0377252487144, 2247.247187252825, 1644.3155338590143, 1096.7811555671415, 967.1794926563396, 2508.3207808332, 877.3036336975632, 1681.1441445551243, 2523.5516212429334, 2382.602109228133, 1937.255120057677, 1703.4629655600288, 1778.824887200208, 1653.0472350297682], "traceback": ["passing", "passing", "passing", "Black took 68 function evals when instructed to use 81", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "Black took 82 function evals when instructed to use 81", "passing", "passing", "passing", "passing", "Black took 82 function evals when instructed to use 81", "Black took 84 function evals when instructed to use 83", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "not yet run", "Black took 81 function evals when instructed to use 81", "Black took 81 function evals when instructed to use 81", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "Black took 81 function evals when instructed to use 81", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "passing", "Traceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/hebo/acq_optimizers/evolution_optimizer.py\", line 119, in optimize\n    res   = minimize(prob, algo, ('n_gen', self.iter), verbose = self.verbose)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/optimize.py\", line 85, in minimize\n    res = algorithm.solve()\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/algorithm.py\", line 226, in solve\n    self._solve(self.problem)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/algorithm.py\", line 321, in _solve\n    self.next()\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/algorithm.py\", line 246, in next\n    self._next()\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/algorithms/genetic_algorithm.py\", line 93, in _next\n    self.off = self.mating.do(self.problem, self.pop, self.n_offsprings, algorithm=self)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/infill.py\", line 40, in do\n    _off = self.eliminate_duplicates.do(_off, pop, off)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/duplicate.py\", line 26, in do\n    pop = pop[~self._do(pop, None, np.full(len(pop), False))]\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/duplicate.py\", line 75, in _do\n    D = self.calc_dist(pop, other)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/duplicate.py\", line 66, in calc_dist\n    D = cdist(X, X)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/util/misc.py\", line 90, in cdist\n    return scipy.spatial.distance.cdist(A, B, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/scipy/spatial/distance.py\", line 2954, in cdist\n    return cdist_fn(XA, XB, out=out, **kwargs)\nValueError: Unsupported dtype object\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/humpday/comparison/eloratings.py\", line 76, in optimizer_game\n    black_best_val, black_best_x, black_feval_count = black(objective, n_trials=n_black_trials,\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/humpday/optimizers/hebocube.py\", line 52, in hebo_sequential_cube\n    return hebo_cube_factory(objective=objective, n_trials=n_trials, n_dim=n_dim, with_count=with_count, n_suggestions=1)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/humpday/optimizers/hebocube.py\", line 39, in hebo_cube_factory\n    rec = opt.suggest(n_suggestions=n_suggestions) # <-- don't change this\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/hebo/optimizers/hebo.py\", line 124, in suggest\n    rec = opt.optimize(initial_suggest = best_x, fix_input = fix_input).drop_duplicates()\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/hebo/acq_optimizers/evolution_optimizer.py\", line 125, in optimize\n    res   = minimize(prob, algo, ('n_gen', self.iter), verbose = self.verbose)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/optimize.py\", line 85, in minimize\n    res = algorithm.solve()\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/algorithm.py\", line 226, in solve\n    self._solve(self.problem)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/algorithm.py\", line 321, in _solve\n    self.next()\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/algorithm.py\", line 246, in next\n    self._next()\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/algorithms/genetic_algorithm.py\", line 93, in _next\n    self.off = self.mating.do(self.problem, self.pop, self.n_offsprings, algorithm=self)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/infill.py\", line 40, in do\n    _off = self.eliminate_duplicates.do(_off, pop, off)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/duplicate.py\", line 26, in do\n    pop = pop[~self._do(pop, None, np.full(len(pop), False))]\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/duplicate.py\", line 75, in _do\n    D = self.calc_dist(pop, other)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/duplicate.py\", line 66, in calc_dist\n    D = cdist(X, X)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/util/misc.py\", line 90, in cdist\n    return scipy.spatial.distance.cdist(A, B, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/scipy/spatial/distance.py\", line 2954, in cdist\n    return cdist_fn(XA, XB, out=out, **kwargs)\nValueError: Unsupported dtype object\n", "Traceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/hebo/acq_optimizers/evolution_optimizer.py\", line 119, in optimize\n    res   = minimize(prob, algo, ('n_gen', self.iter), verbose = self.verbose)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/optimize.py\", line 85, in minimize\n    res = algorithm.solve()\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/algorithm.py\", line 226, in solve\n    self._solve(self.problem)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/algorithm.py\", line 321, in _solve\n    self.next()\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/algorithm.py\", line 246, in next\n    self._next()\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/algorithms/genetic_algorithm.py\", line 93, in _next\n    self.off = self.mating.do(self.problem, self.pop, self.n_offsprings, algorithm=self)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/infill.py\", line 40, in do\n    _off = self.eliminate_duplicates.do(_off, pop, off)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/duplicate.py\", line 26, in do\n    pop = pop[~self._do(pop, None, np.full(len(pop), False))]\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/duplicate.py\", line 75, in _do\n    D = self.calc_dist(pop, other)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/duplicate.py\", line 66, in calc_dist\n    D = cdist(X, X)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/util/misc.py\", line 90, in cdist\n    return scipy.spatial.distance.cdist(A, B, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/scipy/spatial/distance.py\", line 2954, in cdist\n    return cdist_fn(XA, XB, out=out, **kwargs)\nValueError: Unsupported dtype object\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/humpday/comparison/eloratings.py\", line 41, in optimizer_game\n    white_best_val, white_best_x, white_feval_count = white(objective, n_trials=n_white_trials, n_dim=n_dim,\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/humpday/optimizers/hebocube.py\", line 56, in hebo_batch_cube\n    return hebo_cube_factory(objective=objective, n_trials=n_trials, n_dim=n_dim, with_count=with_count, n_suggestions=10)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/humpday/optimizers/hebocube.py\", line 39, in hebo_cube_factory\n    rec = opt.suggest(n_suggestions=n_suggestions) # <-- don't change this\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/hebo/optimizers/hebo.py\", line 124, in suggest\n    rec = opt.optimize(initial_suggest = best_x, fix_input = fix_input).drop_duplicates()\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/hebo/acq_optimizers/evolution_optimizer.py\", line 125, in optimize\n    res   = minimize(prob, algo, ('n_gen', self.iter), verbose = self.verbose)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/optimize.py\", line 85, in minimize\n    res = algorithm.solve()\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/algorithm.py\", line 226, in solve\n    self._solve(self.problem)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/algorithm.py\", line 321, in _solve\n    self.next()\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/algorithm.py\", line 246, in next\n    self._next()\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/algorithms/genetic_algorithm.py\", line 93, in _next\n    self.off = self.mating.do(self.problem, self.pop, self.n_offsprings, algorithm=self)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/infill.py\", line 40, in do\n    _off = self.eliminate_duplicates.do(_off, pop, off)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/duplicate.py\", line 26, in do\n    pop = pop[~self._do(pop, None, np.full(len(pop), False))]\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/duplicate.py\", line 75, in _do\n    D = self.calc_dist(pop, other)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/model/duplicate.py\", line 66, in calc_dist\n    D = cdist(X, X)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/pymoo/util/misc.py\", line 90, in cdist\n    return scipy.spatial.distance.cdist(A, B, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.8.10/x64/lib/python3.8/site-packages/scipy/spatial/distance.py\", line 2954, in cdist\n    return cdist_fn(XA, XB, out=out, **kwargs)\nValueError: Unsupported dtype object\n"], "active": [true, true, true, true, true, false, true, false, false, false, false, false, false, true, true, true, true, true, true, false, false, false, false, false, false, false, false, false, false, false, true, true, true, true, true, true, true, true, true, true, false, false, false, false, false, false, true, true, true, true, true, true, true, true, true, true, false, false, false, false, false, false, false, false, true, true, false, false, false, false]}